Predictor_submission_template.ipynb

```diff
@@ -497,7 +497,7 @@
    "outputs": [],
    "source": [
     "from validation.scenario_generator import get_raw_data, generate_scenario, NPI_COLUMNS\n",
-    "DATA_URL = 'https://raw.githubusercontent.com/OxCGRT/covid-policy-tracker/master/data/OxCGRT_latest.csv'\n",
+    "DATA_FILE = \"validation/tests/fixtures/OxCGRT_latest.csv\"\n",
```

Comment:
Fixing mistake from previous PR for #37 : `get_raw_data` requires a real file name 
Fixed it in the next 2 notebooks too